xwiki-platform-distribution/xwiki-platform-distribution-war-dependencies/pom.xml

```diff
@@ -662,6 +662,18 @@
       <version>${project.version}</version>
     </dependency>
 
+    <!-- Must be installed as a core extension because xwiki-platform-mentions-default must be installed as a core 
```

Comment:
I would get rid of any reference to "installed" to avoid confusion with what we commonly call installed extensions.